### PR TITLE
Add @jonleung's THS specific resources

### LINKED ISSUE #448: Archive The Hack Series landing page/schedule
Related to #442

### PR DESCRIPTION
**Pull request by paked**
Closes #448

### CODE REVIEW

File: the_hack_series/resources/FACILITATION.md [hunk 1]
```diff
@@ -0,0 +1,135 @@
...
+1:10  | _Hack Series Introduction & Icebreakers_
+1:20  | _Introduce Part I_
+1:25  | **Part I: Follow the step-by-step workshop**
+2:30  | **Pizza Time**
+3:00  | _Introduce Part II_
+3:05  | **Part II: Customize your website**
+3:40  | _Collect Demo Links in tlk.io_
+3:45  | **Demos!**
+3:55  | _Collect Feedback_
+4:00  | **Finish**
```
Comment (zachlatta) [inline the_hack_series/resources/FACILITATION.md]: I know that our table format for Markdown files isn't very standardized, but in other places where we use tables we border them with `|`. Can you change this to the following?

```
|   Time | Activity                                     |
| ------ | -------------------------------------------- |
|  11:00 | _Figure out who is doing what_               |
|  12:00 | _Setup_                                      |
|  12:45 | _Play Host & Facilitate Conversation_        |
|   1:10 | _Hack Series Introduction & Icebreakers_     |
|   1:20 | _Introduce Part I_                           |
|   1:25 | **Part I: Follow the step-by-step workshop** |
|   2:30 | **Pizza Time**                               |
|   3:00 | _Introduce Part II_                          |
|   3:05 | **Part II: Customize your website**          |
|   3:40 | _Collect Demo Links in tlk.io_               |
|   3:45 | **Demos!**                                   |
|   3:55 | _Collect Feedback_                           |
|   4:00 | **Finish**                                   |
```

File: the_hack_series/resources/FACILITATION.md [hunk 2]
```diff
@@ -0,0 +1,135 @@
...
+
+Encourage everyone to present.
+
+### 3:55  : Finish
+
+Thank everyone for coming.
+
+# Feedback
+
+- Explain and **_HEAVILY EMPHSIZE** the importance of feedback because it helps
```
Comment (zachlatta) [inline the_hack_series/resources/FACILITATION.md]: Should this be `**_HEAVILY EMPHASIZE_**` instead? It's currently misspelled and there's no closing `_`.

File: the_hack_series/resources/README.md [hunk 3]
```diff
@@ -0,0 +1,56 @@
...
+
+| [![](img/github.png) <br> Join GitHub](github/README.md) |
+|:----------------------------------------------------:|
+
+--
+
+## 3. Choose Your ~~Code Editor~~ Weapon
+
+| [![](img/js_bin.png) <br> Signup for JS Bin](js_bin.md) | OR | [![](img/c9.png) <br> Signup for Cloud9](cloud9.md) |
+|:------------------------------------------------------:|:--:|:------------------------------------------------:| -->
```
Comment (zachlatta) [inline the_hack_series/resources/README.md]: Should this commented out code be removed?

File: the_hack_series/resources/README.md [hunk 4]
```diff
@@ -0,0 +1,56 @@
...
+
+--
```
Comment (zachlatta) [inline the_hack_series/resources/README.md]: Per our styleguide, horizontal line rules should be 80 characters wide.

```
-------------------------------------------------------------------------------
```

File: the_hack_series/resources/README.md [hunk 5]
```diff
@@ -0,0 +1,56 @@
...
+## 2. Get Ready to Build Some Awesome
+
+I want to learn how to build:
+
+| [![](img/portfolio.png) <br> My Own Websites](websites/README.md) |
+|---------------------------------------------------------|
+
+
+<!-- | [![](img/portfolio.png) <br> My Own Websites](websites/README.md) | [![](img/dodge.gif) <br> My Own Video Games](video_games.md) |
+|---------------------------------------------------------|--------------------------------------------------------| -->
```
Comment (zachlatta) [inline the_hack_series/resources/README.md]: Should we remove this commented out code?

File: the_hack_series/resources/README.md [hunk 6]
```diff
@@ -0,0 +1,56 @@
...
+|:------------------------------------------------------:|:--:|:------------------------------------------------:| -->
+
+--
+
+## 2. Get Ready to Build Some Awesome
+
+I want to learn how to build:
+
+| [![](img/portfolio.png) <br> My Own Websites](websites/README.md) |
+|---------------------------------------------------------|
```
Comment (zachlatta) [inline the_hack_series/resources/README.md]: This table should be styled as follows:

```
| [![](img/portfolio.png) <br> My Own Websites](websites/README.md) |
| ----------------------------------------------------------------- |
```

File: the_hack_series/resources/EXTENSION.md [hunk 7]
```diff
@@ -0,0 +1,78 @@
...
+- <a href="http://jsbin.com/gist/c1201dd5f09d99b06c0d?html,css,output" target="_blank">Solution</a>
+
+## Examples of other's websites
+
+- http://aidanb34.github.io/myrepository/
+- http://mnefertiti.github.io/personalwhale/
+- http://animanny.github.io/my_website/
+
+[![](img/bttn_back.png)](https://rawgit.com/jonleung/77f8662577df9c2ff04d/raw/2c8489d490de8e37e89e6fb5b9073f904b6377e0/index.html)
+
```
Comment (zachlatta) [inline the_hack_series/resources/EXTENSION.md]: Is this extra newline intentional?

File: the_hack_series/resources/EXTENSION.md [hunk 8]
```diff
@@ -0,0 +1,78 @@
...
+--
+
+**Change the font**
+
+Try changing the font to something else other than "arial"
+
+- <a href="https://jsbin.com/gist/ddb3f602250d5b8b3cdc?html,css,output" target="_blank">Solution</a>
+
+--
```
Comment (zachlatta) [inline the_hack_series/resources/EXTENSION.md]: Per our styleguide, horizontal rules should be 80 characters wide.

File: the_hack_series/resources/README.md [hunk 9]
```diff
@@ -0,0 +1,56 @@
...
+
+
+<!-- | [![](img/portfolio.png) <br> My Own Websites](websites/README.md) | [![](img/dodge.gif) <br> My Own Video Games](video_games.md) |
+|---------------------------------------------------------|--------------------------------------------------------| -->
+
```
Comment (zachlatta) [inline the_hack_series/resources/README.md]: Is this extra newline intentional?

File: the_hack_series/resources/FACILITATION.md [hunk 10]
```diff
@@ -0,0 +1,135 @@
...
+  us improve.
+- Have hackers submit to this form for feedback
+  http://tinyurl.com/hackseriesfeedback
+
```
Comment (zachlatta) [inline the_hack_series/resources/FACILITATION.md]: Is this extra newline intentional?

**Comment by zachlatta** [comment]
I've left quite a few comments on the styling of this document. Fixing them
may also fix the broken tests :-).

On Fri, Jan 1, 2016 at 2:29 PM, Harrison Shoebridge <
notifications@github.com> wrote:

> Closes #448 <https://github.com/hackclub/meta/issues/448>
> ------------------------------
> You can view, comment on, or merge this pull request online at:
>
>   https://github.com/hackclub/meta/pull/449
> Commit Summary
>
>    - Add @jonleung's THS specific resources
>
> File Changes
>
>    - *A* the_hack_series/resources/EXTENSION.md
>    <https://github.com/hackclub/meta/pull/449/files#diff-0> (78)
>    - *A* the_hack_series/resources/FACILITATION.md
>    <https://github.com/hackclub/meta/pull/449/files#diff-1> (135)
>    - *A* the_hack_series/resources/README.md
>    <https://github.com/hackclub/meta/pull/449/files#diff-2> (56)
>
> Patch Links:
>
>    - https://github.com/hackclub/meta/pull/449.patch
>    - https://github.com/hackclub/meta/pull/449.diff
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/hackclub/meta/pull/449>.
>



-- 
Zach Latta
Hack Club | zach@hackclub.io

**Comment by zachlatta** [comment]
You'll want to rebase `master` into this pull request to get the tests passing :-).

**Comment by paked** [comment]
@MaxWofford this is ready to be merged when hackclub/markcop#60 is merged.

**Comment by MaxWofford** [comment]
Merged hackclub/markcop#60

**Comment by zachlatta** [comment]
:heart:

On Fri, Jan 15, 2016 at 11:32 PM, Max Wofford <notifications@github.com>
wrote:

> Merged #449 <https://github.com/hackclub/meta/pull/449>.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/hackclub/meta/pull/449#event-516884928>.
>



-- 
Zach Latta
http://zachlatta.com
https://github.com/zachlatta
https://twitter.com/zachlatta

**Pull request merged**